.github/workflows/test-python-atlas.yml

```diff
@@ -52,26 +52,11 @@ jobs:
       - name: Start local Atlas
         working-directory: .
         run: bash .github/workflows/start_local_atlas.sh mongodb/mongodb-atlas-local:8.0.15
-      - name: Install mongosh
+      - name: Download crypt shared
         run: |
-          wget -q https://downloads.mongodb.com/compass/mongosh-2.2.10-linux-x64.tgz
-          tar -xzf mongosh-*-linux-x64.tgz
-          sudo cp mongosh-*-linux-x64/bin/mongosh /usr/local/bin/
-          mongosh --version
-      - name: Install mongocryptd from Enterprise tarball
-        run: |
-          curl -sSL -o mongodb-enterprise.tgz "https://downloads.mongodb.com/linux/mongodb-linux-x86_64-enterprise-ubuntu2204-8.0.15.tgz"
-          tar -xzf mongodb-enterprise.tgz
-          sudo cp mongodb-linux-x86_64-enterprise-ubuntu2204-8.0.15/bin/mongocryptd /usr/local/bin/
-      - name: Start mongocryptd
-        run: |
-          nohup mongocryptd --logpath=/tmp/mongocryptd.log &
-      - name: Verify MongoDB installation
-        run: |
-          mongosh --eval 'db.runCommand({ connectionStatus: 1 })'
-      - name: Verify mongocryptd is running
-        run: |
-          pgrep mongocryptd
+          wget https://downloads.mongodb.com/linux/mongo_crypt_shared_v1-linux-aarch64-enterprise-ubuntu2404-8.0.15.tgz
```

Comment:
Probably want to use 8.2.1

Reply:
MongoDB 8.2 is a rapid release with EOL March 30, 2026. Isn't it better to test with MongoDB 8.0 which enterprise are likely to be using?

Reply:
That's fine, there was a test failure that "failed less" (libmongocrypt wire version error vs . aggregation pipeline "let" command error) when I upgraded but IIUC those tests are expected to fail, so when those failures are asserted then 8.0 is probably OK (or 7.0 if you want to support "preview" query types or maybe add a matrix as you suggested somewhere.)